addon/services/raven.js

```diff
@@ -4,7 +4,7 @@ export default RavenService.extend({
     releaseMetaName: 'sentry:revision',
     release: Ember.computed({
         get: function() {
-            return $("meta[name='revision']").attr('content');
+            return $("meta[name='sentry:revision']").attr('content');
```

Comment:
Yes and no. It's apparently (obviously) not supported to set the releaseMetaName for the revisionTagging feature, but that one itself is optional. I.e. you might want to set the revision via the webserver that serves the html (e.g. from redis). It might be nice to make it configurable for the revision tagging too, but that's stuff for another PR.